test/libsolidity/syntaxTests/errors/weird4.sol

```diff
@@ -0,0 +1,9 @@
+error E();
```

Comment:
Some more test ideas:
- Using error outside of a contract where it is defined.
- Using error's selector outside of a contract where it is defined.
- Using `.selector` on an error "instance" (`E().selector`).
- `anonymous` error.
- `using E for L`, `using L for E`.